src/Namotion.Reflection/Namotion.Reflection.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageIconUrl>https://raw.githubusercontent.com/RicoSuter/Namotion.Reflection/master/assets/NuGetIcon.png</PackageIconUrl>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <Description>.NET library with advanced reflection APIs like XML documentation reading, Null Reference Types (C# 8) reflection and string based type checks.</Description>
+    <projectUrl>https://github.com/RicoSuter/Namotion.Reflection</projectUrl>
```

Comment:
Aren't the properies case sensitive?

Reply:
I made this PR in my browser, and I think there was a copy/paste error.

I'll modify it to be identical to what I'm currently using
https://github.com/ScottPlot/ScottPlot/blob/486f2ae38b9a4edb8eeeb1e8bc92c5e1a425ffca/src/ScottPlot/ScottPlot.csproj#L13

And that shows up well on NuGet under the Info section as "project site"
https://www.nuget.org/packages/ScottPlot/